### DISCUSSION TITLE
Συμπληρωματικές συναντήσεις για την ΟΣΣ1

### DISCUSSION THREAD

**Discussion by epidrome** [sdy60/syllabus#90] [Polls]
Για την καλύτερη υποστήριξη της μελέτης και της ΓΕ προτείνονται τακτικές συμπληρωματικές ΟΣΣ για τα δύο ΣΚ πριν την τυπική που ορίζει το ΕΑΠ

**Comment by Maurphal** [comment]
Καλημέρα! καλή αρχή. Πότε θα γίνει η συμπληρωματική ΟΣΣ?

**Reply by epidrome** [reply]
Για την ώρα φαίνεται να κερδίζει το Σάββατο 7μμ, ας περιμένουμε μέχρι την Πέμπτη το βράδυ να ψηφίσουν περισσότεροι πριν βγει μια ανακοίνωση με το αποτέλεσμα.

**Comment by nikos7887** [comment]
Καλημέρα σας. Καλό ακαδημαϊκό έτος. Μήπως υπάρχει η δυνατότητα να διεξαχθεί η πρώτη συμπληρωματική ΟΣΣ πρωινές ώρες για ΣΚ;

Σας ευχαριστώ.

**Reply by epidrome** [reply]
Ενημερώθηκε με δύο πρωινές επιλογές, πάντως 9 άτομα έχουν ήδη ψηφίσει για αύριο Σάββατο 7μμ οπότε είναι λίγο δύσκολο να αλλάξει τώρα το αποτέλεσμα αλλά ίσως αλλάξει για το επόμενο ΣΚ αν θέλετε

**Reply by std159312** [reply]
Καλημέρα . Ευχαριστούμε
